dali/operators/audio/preemphasis_filter_op.h

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_AUDIO_PREEMPHASIS_FILTER_OP_H_
+#define DALI_OPERATORS_AUDIO_PREEMPHASIS_FILTER_OP_H_
+
+#include <random>
+#include <vector>
+#include "dali/core/convert.h"
+#include "dali/pipeline/operator/operator.h"
+
+namespace dali {
+namespace detail {
+
+/**
+ * Names of arguments
+ */
+const std::string kCoeff = "preemph_coeff";  // NOLINT
+const std::string kDtype = "dtype";          // NOLINT
```

Comment:
`dtype` seems to be reused in many places, why not put it in some common operator header.